Gegno ship balance changes

Amazinite opened the pull request.

Amazinite committed:
Halite

-22% cost: the Halite has very little space, and so it should be on the cheaper end of fighters in the game, while previously it costed as much as a Dagger.

+60 shields, -80 hull: keep shields and hull at multiples of 100.

-1 cargo space: funny on the Dunite, not needed here.

+0.19 heat dissipation: make the Halite's heat diss higher end instead of lower end, since it has so little space.

Amazinite committed:
Dunite

+40 hull: keep hull at multiple of 100

Amazinite committed:
Shale, Slate, Mica

Better differentiate the three transports

Shale:
* +40 hull: round out hull.
* -28 mass: bring mass a bit more in line with similarly sized human ships.

Slate:
* +175 hull: round out hull while making it a bit more tanky than the Shale.
* -3 bunks, +21 cargo: lean more into freighter role.
* Changed category to Light Freighter.
* -5 mass: bring more in line with Shale.

Mica:
* +1 bunk, -6 cargo: lean more into transport role.
* -45 mass: bring more in line with Shale.

Amazinite committed:
Gypsum

-500 hull: just a little weaker to better fit its size.

-6 bunks: less spare space for a freighter.

-20 mass: just a little lighter to better fit its size.

+100 fuel: give the Gypsum enough fuel to jump three systems out and back from a spaceport.

-60 cargo: 220 is actually pretty high end. Make the Gypsum more mid-range to leave room for other Gegno freighters.

Amazinite committed:
Tridymite

-1100 hull: make it less tanky. While it is described as a scout ship, it should be using its speed to avoid damage, not be tanky. Aside from a Dunite, this is actually already the fastest Gegno ship.

-300k cost: offset for the hull reduction.

+200 fuel capacity: give it a bit more "scouting range."

-40 outfit space: trimming a lot of excess space

-40 mass: give it a bit more maneuverability.

Amazinite committed:
Coesite

-600 hull: bring more in line with Tridymite.

-30 mass: give more maneuverability.

+100 fuel: give more jump range.

Change loadout: provide a bit more speed so that it's more on-par with an Eclogite.

Amazinite committed:
Eclogite

-3 bunks: bring carrying capacity below Coesite

Amazinite committed:
Dolomite

Make the cost not the exact same as the Gypsum and more in line with similarly sized ships.

-2300 hull: bring more in line with similarly sized ships.

-9 required crew, -1 bunks: bring required crew more in line with other heavy freighters of similar size, while keeping larger bunk space.

-30 mass, -1.3 drag: make a bit more maneuverable.

-34 outfit space: trimming excess.

-0.1 heat dissipation: bring more in line with similarly sized ships.

Changed outfits: give enough energy to cover engines.

Amazinite committed:
Conglomerate

You know what, I don't really have time to type all the changes out right now. You can see them. Justifications for changes are basically the same as the changes to other ships.

Amazinite committed:
Protolith

Ditto, and also not make this just a second Bactrian, but a similar ship which is more cargo heavy than bunk heavy.

Saugia commented on data/gegno/gegno ships.txt:
```diff
@@ -766,19 +768,19 @@ ship "Protolith"
 		licenses
 			"Vi Lord"
-		"cost" 17600000
+		"cost" 168820000
```
Everything looks pretty good so far. The only question I have is this, did you put an extra 0 or something by accident? The Conglomerate, a somewhat similarly-sized vessel, is 5,909,000, while this is 168,820,000. The Bactrian is 17,600,000. Is this supposed to be 16,882,000?
```suggestion
		"cost" 16882000
```

Amazinite replied:
Per the spreadsheet, it was actually an accidental extra 8.

Saugia replied:
Dang it ol' spreadsheet.

Amazinite committed:
Fix Protolith cost

Amazinite committed:
Augen

-10 drag: just lowering the ship's max speed. Since it's a superheavy, things don't really need to be super nailed down aside from it being big and slow.

Amazinite committed:
Schist, Felsic

Adjust various stats to be more in line with Dolomite (hull, crew, bunks, cost).

Trimmed excess engine space on Schist.

Bumped some Felsic up a bit more above the Schist, changing description to match.

Amazinite committed:
Diorite

Big rework on this one. The description was somewhat contradictory: so this things is big enough to mount two ballistic cannons, but was also described as "maneuverable"? Split the ballistic cannon usage off into a variant.

Amazinite committed:
Hornfel

Trimmed some excess outfit and weapon space. Adjusted other stats to better fit size.

Amazinite committed:
Gneiss

Actually already pretty good where it was. Upped the hull and dropped the mass and drag a bit.

Amazinite committed:
Merge branch 'pr/6610' into gegno-balance

Amazinite committed:
Feldspar, Epidote

More of the same.

Brought down heat dissipations, which were way too high for ships these size.

Give Epidote enough outfit space to fit two artillery turrets if it wanted.
